Test/allocated1/dafny0/CompilationErrors.dfy.expect

```diff
@@ -1,23 +1,25 @@
+CompilationErrors.dfy(6,13): Warning: Method _module._default.M has no body. Add the {:axiom} attribute to it to suppress this warning
+CompilationErrors.dfy(7,7): Warning: Method _module._default.P has no body. Add the {:axiom} attribute to it to suppress this warning
+CompilationErrors.dfy(19,9): Warning: Function _module._default.F has no body. Add the {:axiom} attribute to it to suppress this warning
+CompilationErrors.dfy(20,16): Warning: Function _module._default.H has no body. Add the {:axiom} attribute to it to suppress this warning
+CompilationErrors.dfy(23,2): Warning: assume statement has no {:axiom} annotation
+CompilationErrors.dfy(27,2): Warning: assume statement has no {:axiom} annotation
+CompilationErrors.dfy(33,2): Warning: assume statement has no {:axiom} annotation
+CompilationErrors.dfy(43,6): Warning: assume statement has no {:axiom} annotation
+CompilationErrors.dfy(55,8): Warning: assume statement has no {:axiom} annotation
+CompilationErrors.dfy(60,16): Warning: assume keyword in assign-such-that statement has no {:axiom} annotation
 CompilationErrors.dfy(71,2): Warning: note, this loop has no body (loop frame: x)
 CompilationErrors.dfy(73,2): Warning: note, this loop has no body (loop frame: i, x)
 CompilationErrors.dfy(83,4): Warning: note, this loop has no body (loop frame: x)
 CompilationErrors.dfy(85,4): Warning: note, this loop has no body (loop frame: i, x)
+CompilationErrors.dfy(12,6): Warning: assume statement has no {:axiom} annotation
```

Comment:
For the near future: it would be nice to have a warning code attached to make it easier to look up the explanation for why this is an issue.

For now: could we add an FAQ entry? https://github.com/dafny-lang/dafny/tree/master/docs/HowToFAQ. Even just one with all the new error messages with a link to the relevant reference manual section would be great.

Reply:
> For the near future: it would be nice to have a warning code attached to make it easier to look up the explanation for why this is an issue.

IMO the mechanism needs some refactoring before we start using it more.

> For now: could we add an FAQ entry? https://github.com/dafny-lang/dafny/tree/master/docs/HowToFAQ. Even just one with all the new error messages with a link to the relevant reference manual section would be great.

Could you give me a sample of what you're looking for? I'd like to add a detailed description that can be shown on hover, but as a I mentioned I think that feature needs a little work.